authnid/make_app.py

```diff
@@ -61,6 +61,8 @@ def _make_crl_validator(app):
     app.crl_validator = Validator(
         roots=[app.config["CA_CHAIN"]], crl_locations=crl_locations
     )
+    for e in app.crl_validator.errors:
+        app.logger.info(e)
```

Comment:
Why at the `info` level? These are errors, yes?

Reply:
True. Will fix.